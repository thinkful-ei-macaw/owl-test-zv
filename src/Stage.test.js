import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Stage from './Stage';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Stage />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
  const tree = renderer
    .create(<Stage />)
    .toJSON();
  expect(tree).toMatchSnapshot();  
});
