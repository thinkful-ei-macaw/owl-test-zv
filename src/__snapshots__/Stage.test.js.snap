// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="Stage"
>
  <div>
    <div>
      Koren Templeton
      <img
        alt="user icon"
        src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
      />
    </div>
    <div>
      Ashla Attwool
      <img
        alt="user icon"
        src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
      />
    </div>
  </div>
</div>
`;
